gitpod-io/website
# Add a blog post generator.

## Description

See #1599.

## Related Issue(s)

Close #1599

## How to test

1. Start a new workspace
2. Run `npm run generate:blog-post`
3. Answer all prompts
4. Run `npm run dev`
5. Navigate to `/blog` and make sure the new blog post is listed

## Release Notes

```release-note
None
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1600"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1600"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
